### ISSUE TITLE
Suppport Mozilla Firefox browser

### ISSUE THREAD

**Issue by immattdavison** [immattdavison/NoMoreDomains#58]
create a conversion of this extension for mozilla firefox

**Comment by gavinroderick** [comment]
hey, I'd like to pick this up if that's ok?

**Comment by immattdavison** [comment]
> hey, I'd like to pick this up if that's ok? 

Assigned!

**Comment by gavinroderick** [comment]
damn that was speedy! 🏎️

**Comment by immattdavison** [comment]
> damn that was speedy! 🏎️  
> 
> 
> 
> 

🏎️💨

**Comment by codingfarhan** [comment]
Hi, what's the status? Can I help in any way? I'm open to fixing any issue in the project so please let me know.

**Comment by gavinroderick** [comment]
Hey @codingfarhan I picked this up and started looking - i think it's probably a bit above what I have capacity to do rn, happy if you want to pick it up and try it! 

cc. @immattdavison

**Comment by immattdavison** [comment]
Hey @gavinroderick there is already a PR for this, it just needs rearranging so that MV2 and MV3 versions are separated, I can work on it over the weekend unless you want to?

**Comment by gavinroderick** [comment]
yeah I looked at the PR, I'll be honest I don't have any experience with extensions and figured this would be a nice place to start! works been super draining this week so i don't think i'll be working on it, sorry!

**Comment by immattdavison** [comment]
> yeah I looked at the PR, I'll be honest I don't have any experience with extensions and figured this would be a nice place to start! works been super draining this week so i don't think i'll be working on it, sorry!

That's fair enough, thanks for your help!

**Comment by codingfarhan** [comment]
Late here, but I would love to pick up on the issue. @immattdavison can I still work on this?

**Comment by immattdavison** [comment]
> Late here, but I would love to pick up on the issue. @immattdavison can I still work on this?

For sure - please not FireFox requires MV2 - the best option would be to put all current files into a folder called MV3 and make another version with the same functions in a folder named MV2 of course leave README and other similar files in the root dir

**Comment by immattdavison** [comment]
@codingfarhan #61 may be a good starting point!

**Comment by codingfarhan** [comment]
> @codingfarhan #61 may be a good starting point!

Thanks, I'll get right into it.

**Comment by immattdavison** [comment]
> > @codingfarhan #61 may be a good starting point!
> 
> 
> 
> Thanks, I'll get right into it.

Thanks! Looking forward to your PR!

**Comment by codingfarhan** [comment]
Facing difficulty implementing MV2 for Firefox, I may not be able to do it. Sorry guys. Please look over it @immattdavison
